pkgs/applications/networking/browsers/firefox-pmos-mobile/default.nix

```diff
@@ -0,0 +1,70 @@
+{ browser
+, lib
+, fetchFromGitLab
+, stdenv
+, wrapFirefox
+, firefoxLibName ? "firefox"
+, forceWayland ? true
+, keepHomepage ? true
+, distroName ? "NixOS"
+}:
+let
+  version = "2.0.1";
+
+  mobile-config-firefox = stdenv.mkDerivation {
+    name = "mobile-config-firefox";
+    src = fetchFromGitLab {
+      owner = "postmarketOS";
+      repo = "mobile-config-firefox";
+      rev = version;
+      sha256 = "0kqq31h4qdqbn8y65jz27rsn7v85q3bdprdx1fb0c9s3g9lh5zqx";
+    };
+
+    makeFlags = lib.optional (distroName != null) "DISTRO=${distroName}";
```

Comment:
I don't see a reason to make this overridable. If people want to change it, they should use `overrideAttrs`.